app/scripts/HiGlassComponent.js

```diff
@@ -3890,7 +3890,6 @@ class HiGlassComponent extends React.Component {
       }
 
       if (!v.initialXDomain) {
-        console.warn('No initialXDomain provided in the view config.');
```

Comment:
I've never once found this helpful. It's just annoying. If no initialXDomain is provided, higlass automatically zooms to the data extent.

Reply:
Nice! Blast this line of code into the void!1!

Minor comment: In the next line we're setting `initialXDomain` to `[0, 100]` which is not the data extent and can cause confusion. If the value assignment is meaningless (i.e., higlass does zoom to the data extent at some point) then I think we should fix the next line too.

Reply:
We need to assign something to `initialXDomain` otherwise all scaling and zooming will be completely broken.

Reply:
`zoomToDataExtent` is asynchronous and depends on tracks being present. If that line isn't there, then we'll end up with a useless component.